gstools/variogram/variogram.py

```diff
@@ -75,6 +77,17 @@ def vario_estimate_unstructured(
         the spatially distributed data
     bin_edges : :class:`numpy.ndarray`
         the bins on which the variogram will be calculated
+    angles : :class:`numpy.ndarray`
+        the angles of the main axis to calculate the variogram for in radians
+        angle definitions from ISO standard 80000-2:2009 
+        for 1d this parameter will have no effect at all
+        for 2d supply one angle which is azimuth φ (ccw from +x in xy plane)
+        for 3d supply two angles which are inclination θ (cw from +z)
+        and azimuth φ (ccw from +x in xy plane)
+    angles_tol : :float
```

Comment:
please change to
angles_tol : :class:`float`

Reply:
will do

Reply:
for my own records: DONE